<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polkit Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0,5,0">
    <item row="2" column="0">
     <widget class="Line" name="v_separator">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="policies_tree_view">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="filter_line_edit">
      <property name="placeholderText">
       <string>Filter</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="priveleges_h_layout" stretch="1,0,1">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="explicit_v_layout" stretch="0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="explicit_label">
          <property name="text">
           <string>Explicit Priveleges</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="explicit_inner_h_layout">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="explicit_table_view"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="explicit_buttons_v_layout">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="add_button">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove_button">
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/resources.qrc">
                <normaloff>:/resources/remove.png</normaloff>:/resources/remove.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="h_separator">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="implicit_v_layout" stretch="1,4,4,4,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="implicit_label">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Implicit Priveleges</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="any_user_h_layout" stretch="1,2">
          <property name="spacing">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="any_user_label">
            <property name="text">
             <string>Any user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="any_user_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="inactive_sess_h_layout" stretch="1,2">
          <property name="spacing">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="inactive_sess_label">
            <property name="text">
             <string>Inactive session</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="inactive_sess_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="active_sess_h_layout" stretch="1,2">
          <property name="spacing">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="active_sess_label">
            <property name="text">
             <string>Active session</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="active_sess_combobox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="save_button_h_layout">
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/resources.qrc">
              <normaloff>:/resources/save.png</normaloff>:/resources/save.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="Line" name="bottom_line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="action_expand_tree"/>
    <addaction name="action_collapse_tree"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/resources/quit.png</normaloff>:/resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/resources/info.png</normaloff>:/resources/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_expand_tree">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/resources/expand.png</normaloff>:/resources/expand.png</iconset>
   </property>
   <property name="text">
    <string>Expand tree</string>
   </property>
  </action>
  <action name="action_collapse_tree">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/resources/collapse.png</normaloff>:/resources/collapse.png</iconset>
   </property>
   <property name="text">
    <string>Collapse tree</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
